DigitalLearningSolutions.Web.Tests/Helpers/ProfessionalRegistrationNumberHelperTests.cs

```diff
@@ -100,6 +102,7 @@ public void ValidateProfessionalRegistrationNumber_sets_error_when_hasPrn_is_not
             );
 
             // Then
+            const string expectedError = "Select your professional registration number status.";

```

Comment:
Is there any reason not to inline this? No need for re-review, do what you think is best.

Reply:
There is a habit of explicitly declaring test input and output where it appears to matter more for clarity on test scope.
But to be fair these are quite simplistic tests so we can inline this.
Sorted this now.